packages/lit-element-starter-js/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "lit-element-starter-js",
+  "version": "0.0.0",
+  "description": "A simple web component",
+  "main": "my-element.js",
+  "module": "my-element.js",
+  "type": "module",
+  "scripts": {
+    "build": "echo \"This is not a TypeScript project, so no need to build.\"",
+    "lint": "npm run lint:lit-analyzer && npm run lint:eslint",
+    "lint:eslint": "eslint '**/*.js'",
+    "lint:lit-analyzer": "lit-analyzer my-element.js",
+    "format": "prettier \"**/*.{cjs,html,js,json,md,ts}\" --ignore-path ./.eslintignore --write",
+    "docs": "npm run docs:clean && npm run analyze && npm run docs:build && npm run docs:assets && npm run docs:gen",
+    "docs:clean": "rimraf docs",
+    "docs:gen": "eleventy --config=.eleventy.cjs",
+    "docs:gen:watch": "eleventy --config=.eleventy.cjs --watch",
+    "docs:build": "rollup -c --file docs/my-element.bundled.js",
+    "docs:assets": "cp node_modules/prismjs/themes/prism-okaidia.css docs/",
+    "docs:serve": "wds --root-dir=docs --node-resolve --watch",
+    "analyze": "wca analyze \"**/*.js\" --outFile custom-elements.json",
+    "serve": "wds --node-resolve --watch",
+    "test": "wtr",
+    "test:watch": "npm run test -- --watch",
+    "checksize": "rollup -c ; cat my-element.bundled.js | gzip -9 | wc -c ; rm my-element.bundled.js"
+  },
+  "keywords": [
+    "web-components",
+    "lit-element",
+    "javascript"
+  ],
+  "author": "The Polymer Authors",
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "lit-element": "^3.0.0-pre.1"
+  },
+  "devDependencies": {
+    "@11ty/eleventy": "^0.10.0",
+    "@11ty/eleventy-plugin-syntaxhighlight": "^3.0.1",
+    "@esm-bundle/chai": "^4.1.5",
+    "@open-wc/testing": "^2.5.32",
+    "@open-wc/testing-karma": "^3.3.11",
+    "@rollup/plugin-node-resolve": "^9.0.0",
+    "@rollup/plugin-replace": "^2.3.3",
+    "@types/mocha": "^7.0.2",
+    "@web/dev-server": "0.0.29",
+    "@web/dev-server-legacy": "^0.1.4",
+    "@web/dev-server-rollup": "^0.2.7",
+    "@web/test-runner": "^0.10.2",
+    "@web/test-runner-mocha": "^0.3.5",
+    "@web/test-runner-playwright": "^0.5.7",
+    "@webcomponents/webcomponentsjs": "^2.5.0",
+    "babel-eslint": "^10.1.0",
+    "deepmerge": "^4.2.2",
+    "eslint": "^6.8.0",
+    "lit-analyzer": "^1.1.10",
+    "lit-html": "^2.0.0-pre.4",
```

Comment:
I wonder what uses lit-html directly, but only in dev? I think this could be removed

Reply:
Removed